app/Http/Controllers/API/Posts/GeoJSONController.php

```diff
@@ -40,7 +40,8 @@ public function prepBoundingBox(Request $request)
     protected function getFilters(Request $request)
     {
         return parent::getFilters($request) + [
-            'include_types' => ['point', 'geometry']
+            'include_types' => ['point', 'geometry'],
+            'output_core_post' => true
```

Comment:
may be good to add a comment on this new filter's meaning

Reply:
I added it here: 

https://github.com/ushahidi/platform/pull/3907/files#diff-15be46c5155c0769e0fb9dd020f43de8R493

since that's where all the filters are anyway... Shall I duplicate it?

Reply:
oh, I think that's fine then. Probably something we should have in a static const but not a worry.